anet.yml

```diff
@@ -461,6 +461,7 @@ dictionary:
   automaticallyInactivateUsers:
     emailRemindersDaysPrior: [15, 30, 45]
     ignoredDomainNames: []
+    checkIntervalInSecs: 3600  # 60 * 60
```

Comment:
Is there a point in checking every hour? As the end of tour date is at a level of granularity of a day, is there a benefit of running this task more than once a day?

Reply:
You are correct, I changed it to 86400 (1 day)